resources/js/Pages/Home.vue

```diff
@@ -189,6 +189,7 @@ function showRandom(): void{
     display: flex;
     justify-content: space-between;
     align-items: center;
+    gap: var(--dimension-spacing-small);
```

Comment:
The gap is for this spec, right? 
![image](https://github.com/wmde/wikidata-mismatch-finder/assets/3005239/9b112952-a980-4022-9632-a953a23a2c35)

Reply:
For this part: 
![image](https://github.com/wmde/wikidata-mismatch-finder/assets/7420952/70b260ae-0baa-45a6-841c-14c0de17d1ab)

But I realized I skipped the gap in `Results.vue` so I should add that too